client/packages/common/src/intl/locales/en/common.json

```diff
@@ -232,8 +231,7 @@
   "label.pack": "Pack",
   "label.packs-of_one": "pack of",
   "label.packs-of_other": "packs of",
-  "label.pack-quantity": "Pack Qty",
-  "label.pack-quantity-issued": "Pack Qty Issued",
+  "label.pack-quantity-issued": "# of packs Issued",
```

Comment:
I wonder if we need to remove pack-quantity from other translations ?

Reply:
e0d890a